go.sum

```diff
@@ -719,8 +719,6 @@ github.com/hashicorp/vault-plugin-auth-jwt v0.10.1 h1:7hvGSiICXpmp7Ras5glxVVxTDg
 github.com/hashicorp/vault-plugin-auth-jwt v0.10.1/go.mod h1:3KxfehLIM7zH19+O8jHJ/QJsLGRzSKRqjsesOJmBuoI=
 github.com/hashicorp/vault-plugin-auth-kerberos v0.4.0 h1:7M7/DbFsUoOMBd2/R48ZNj4PM3Gdsg0dGcbMOdt5z1Q=
 github.com/hashicorp/vault-plugin-auth-kerberos v0.4.0/go.mod h1:h+7pLm4Z2EeKHOGPefX0bGzdUQCMBUlvM/BpSMNgTFw=
-github.com/hashicorp/vault-plugin-auth-kubernetes v0.11.1-0.20210921194437-e5af6ccd8add h1:Spwfyp4obQ6MhXWCsYHiAlNsehb8PCVciF1vMZqn3so=
```

Comment:
This looks unrelated to the dep updates. Can we rebase/merge `origin/main` once more? I just [merged a PR](https://github.com/hashicorp/vault/pull/12712) that synced up the deps recently which should take this change out of the PR.

I don't see other deps being pulled in, but can we do the dep update without the `-u` option just in case, so that we don't end up pulling transitive dep updates?
```
$ go get github.com/lib/pq
```

Reply:
This came from running `go mod tidy` and occurs on the head of main still for me:
```
~/g/h/vault (main)> git fetch
~/g/h/vault (main)> git merge --ff-only
Already up to date.
~/g/h/vault (main)> go mod tidy
~/g/h/vault (main)> git diff
diff --git a/go.sum b/go.sum
index 70ded0821..70754edc1 100644
--- a/go.sum
+++ b/go.sum
@@ -719,8 +719,6 @@ github.com/hashicorp/vault-plugin-auth-jwt v0.10.1 h1:7hvGSiICXpmp7Ras5glxVVxTDg
 github.com/hashicorp/vault-plugin-auth-jwt v0.10.1/go.mod h1:3KxfehLIM7zH19+O8jHJ/QJsLGRzSKRqjsesOJmBuoI=
 github.com/hashicorp/vault-plugin-auth-kerberos v0.4.0 h1:7M7/DbFsUoOMBd2/R48ZNj4PM3Gdsg0dGcbMOdt5z1Q=
 github.com/hashicorp/vault-plugin-auth-kerberos v0.4.0/go.mod h1:h+7pLm4Z2EeKHOGPefX0bGzdUQCMBUlvM/BpSMNgTFw=
-github.com/hashicorp/vault-plugin-auth-kubernetes v0.11.1-0.20210921194437-e5af6ccd8add h1:Spwfyp4obQ6MhXWCsYHiAlNsehb8PCVciF1vMZqn3so=
-github.com/hashicorp/vault-plugin-auth-kubernetes v0.11.1-0.20210921194437-e5af6ccd8add/go.mod h1:Q13bq4paoPWW+bsSq2seyiLPQkFl5vrb+vIwwLDlQ8M=
 github.com/hashicorp/vault-plugin-auth-kubernetes v0.11.1-0.20210929181055-821e911b1751 h1:wICfRtupLijLDjQ/8GGnEOvpDzxGK1pwd1OQBZFQOr0=
 github.com/hashicorp/vault-plugin-auth-kubernetes v0.11.1-0.20210929181055-821e911b1751/go.mod h1:Q13bq4paoPWW+bsSq2seyiLPQkFl5vrb+vIwwLDlQ8M=
 github.com/hashicorp/vault-plugin-auth-oci v0.8.0 h1:qYtVYsQlVnqqlCVqZ+CAiFEXuYJqUQCuqcWQVELybZY=
```

I can rewrite the commits to drop the `go mod tidy` but had added that call based on this comment's suggestion: https://github.com/hashicorp/vault/pull/12413#discussion_r717217156

Reply:
It's okay. It's just removing the older checksums of the `github.com/hashicorp/vault-plugin-auth-kubernetes` package. You can see the latest version of the plugin that's in use remains just below these changes:

```
github.com/hashicorp/vault-plugin-auth-kubernetes v0.11.1-0.20210929181055-821e911b1751 h1:wICfRtupLijLDjQ/8GGnEOvpDzxGK1pwd1OQBZFQOr0=
```

Reply:
What version of `go` are you on?

Reply:
```
go version
go version go1.17.1 darwin/amd64
```
it is honoring the go1.16 directive in the go.mod file and not making 2 requires blocks ala 1.17 -- if it is an issue let me know and I'll remove commit d43574b

Reply:
Yep, we should be running `go mod tidy`. It's odd that this removal didn't apply already in my recently commit which I also ran `go mod tidy` against. I was hoping to clean this up so that it'd make backporting and conflict resolution a little easier, but we can also address this on our end.

Can we also include a changelog file that mentions the bug fix as part of this PR? [Here](https://github.com/hashicorp/vault/blob/main/changelog/12622.txt) is a sample of the formatting for a changelog entry file (the name of the file should be the PR number). Something along the lines of:

```
storage/postgres: Update postgres library to properly clean up connection pool on write errors
database/postgres: Update postgres library to properly clean up connection pool on write errors
```

Sorry for the numerous back-and-forths on this!

Reply:
added `changelog/12413.txt` in 2b37f5787